### DISCUSSION TITLE
Contributions and Activity overview on Profile don't match

### DISCUSSION THREAD

**Discussion by javali93** [community/community#4437] [Other Feature Feedback, Questions, & Ideas]
The activity overview graph only shows the last or latest months contributions plotted by default and even when we select a particular year instead of the whole year's contributions

- Showing only for July 1st

<img width="1418" alt="Screenshot 2021-07-01 at 1 04 04 PM" src="https://user-images.githubusercontent.com/26891686/124086219-763e8a80-da6e-11eb-8c63-30c4fb32c512.png">

- Showing only for July 1st when selected 2021

<img width="1493" alt="Screenshot 2021-07-01 at 12 49 11 PM" src="https://user-images.githubusercontent.com/26891686/124086234-7b033e80-da6e-11eb-8f57-fbc000abab15.png">

- Showing only for Dec when selected 2020

<img width="1439" alt="Screenshot 2021-07-01 at 12 49 31 PM" src="https://user-images.githubusercontent.com/26891686/124086241-7c346b80-da6e-11eb-8b70-3dfe0320132c.png">

**Comment by jfullstackdev** [comment]
yup, that's right

but I'm using this external service so I can have the overview

https://github.com/anuraghazra/github-readme-stats

and many added services, like extra pins

and much more, if you are really into it, you can contribute to that repo for your added features

this is my sample overview

![image](https://user-images.githubusercontent.com/47092464/196042086-680f397f-279e-492c-aff2-6894529d4c14.png)
